ApiBundle/Controller/ContentController.php

```diff
@@ -101,10 +101,32 @@ public function showOrCreateAction(Request $request, $contentId)
     public function listAction(Request $request)
     {
         $contentType = $request->get('content_type');
+        $columns = $request->get('columns');
```

Comment:
Are you sure?

You could give as a parameter the name of the method in the repository for example